tests/conftest.py

```diff
@@ -237,26 +259,43 @@ def provenance_preprocessing(code_version):
         session.commit()
 
 
-@pytest.fixture
-def archive(test_config):
+@pytest.fixture(scope="session", autouse=True)
+def archive_path(test_config):
+    if test_config.value('archive.local_read_dir', None) is not None:
+        archivebase = test_config.value('archive.local_read_dir')
+    elif os.getenv('SEECHANGE_TEST_ARCHIVE_DIR') is not None:
+        archivebase = os.getenv('SEECHANGE_TEST_ARCHIVE_DIR')
+    else:
+        raise ValueError('No archive.local_read_dir in config, and no SEECHANGE_TEST_ARCHIVE_DIR env variable set')
+
+    # archive.path_base is usually /test
+    archivebase = pathlib.Path(os.path.join(archivebase, test_config.value('archive.path_base')))
```

Comment:
```suggestion
    archivebase = pathlib.Path(archivebase) / test_config.value('archive.path_base')
```

pathlib.Path overloads the / operator to append subdirectories and files.  (In general, when using pathlib, you nearly never need to use any of the os.path functions.)

Reply:
I know, we have very different ways of working with paths. I will merge it and then make sure this part makes sense.

Reply:
I like the pathlib library, and have never internalized the os.path functions.

I think it's fine that we have both ways of doing things in the code, but there's no need to mix the two in a single line...